yarn-project/circuits.js/src/structs/shared.ts

```diff
@@ -31,15 +47,24 @@ export class UInt8Vector {
   }
 }
 
+/**
+ * A type alias for a 32-bit unsigned integer.
+ */
 export type UInt32 = number;
 
 /**
  * Affine element of a group, composed of two elements in Fq.
- * cpp/barretenberg/cpp/src/aztec/ecc/groups/affine_element.hpp
- * cpp/barretenberg/cpp/src/aztec/ecc/curves/bn254/g1.hpp
+ * Cpp/barretenberg/cpp/src/aztec/ecc/groups/affine_element.hpp
+ * cpp/barretenberg/cpp/src/aztec/ecc/curves/bn254/g1.hpp.
```

Comment:
We need to kill that "sentences end in period" rule, it backfires more than it helps

Reply:
Linter sneaked in this change. I've reverted in f145f3dbd1852e193b1e520667534464434b72df and disabled the rule for the file.